holoviews/plotting/bokeh/callbacks.py

```diff
@@ -663,15 +665,15 @@ def _update_selection_event(self, event):
         self._selection_event = event
         self._processed_event = not event.final
         if event.final and self._skipped_partial_event:
-            self._process_selection_event()
+            async_execute(self._process_selection_event)
             self._skipped_partial_event = False
```

Comment:
Is this a safe change?

`self._skipped_partial_event` will be set to False immediately and not after the method has been called.

Reply:
I'm not sure, just wanted to flag that you are making a potential change.

I'm not sure you can move it inside either, as other calls to the method that previously did not reset it will now do it.

Reply:
Well tests seem to point to lasso select failing
https://github.com/holoviz/holoviews/actions/runs/11056395453/job/30717855703?pr=6390

Reply:
Ok I think wrapping it in its own helper function should keep old behavior